migration_original/ODS1Stage/tables/Show/SOLRProviderRedirect/spu_translated_SOLRProviderRedirect.sql

```diff
@@ -0,0 +1,366 @@
+
+-- sp_load_solrproviderredirect (validated in snowflake)
+
+
+-- 1. Update show.solrproviderredirect where the columns are different 
+-- 2. Insert into show.solrproviderredirect where the providerid is null
+-- 3. Delete from show.solrproviderredirect if the provider is in show.solrprovider
+-- 4. DELTA: Insert into show.solrproviderredirect where the providerid is null
+-- 5. DELTA: Update show.solrproviderredirect where URLs are null
+
+CREATE OR REPLACE PROCEDURE SHOW.SP_LOAD_SOLRPROVIDERREDIRECT(IsProviderDeltaProcessing BOOLEAN)
```

Comment:
Add database to the procesure name.
```suggestion
CREATE OR REPLACE PROCEDURE ODS1STAGE.SHOW.SP_LOAD_SOLRPROVIDERREDIRECT(IsProviderDeltaProcessing BOOLEAN)
```